pom.xml

```diff
@@ -31,7 +31,7 @@ under the License.
 
     <groupId>com.bazaarvoice.maven.plugins</groupId>
     <artifactId>cassandra-maven-plugin</artifactId>
-    <version>2.2.4-6-SNAPSHOT</version>
+    <version>2.2.19-1-SNAPSHOT</version>
```

Comment:
this will be handled by maven release plugin in jenkins

Reply:
reverted to 2.2.4-6